onnxruntime/core/framework/session_options.h

```diff
@@ -55,11 +56,11 @@ struct SessionOptions {
   TransformerLevel graph_optimization_level = TransformerLevel::Level3;
 
   // controls the size of the thread pool used to parallelize the execution of tasks within individual nodes (ops)
-  int intra_op_num_threads = 0;
+  OrtThreadPoolParams intra_op_param;
```

Comment:
1. Do we plan to allow users to set params other than threadpool size via our APIs?
2. If so, we should update the perf turning doc and other documentation.

Reply:
Yes. It can be in a different PR. We can discuss which options we want to expose and how.